### PR TITLE
Tool tips

### LINKED ISSUE #1478: Add tooltips to eCR Viewer Fields [2/2]
## Action Requested
Add tooltips for the following 5 fields:

## Acceptance Criteria
When users hover over the 5 field names above, the should see a tooltip displaying the content specified in the tooltip table (attached in Design section below)

## Testing
(not sure)

## Background & Context
Some of the fields on the eCR viewer are not self explanatory for all users, so we're adding tooltips to help users understand what the data is and where it comes from

## Designs
I didn't mock up designs for this since it will just be standard USWDS component usage, so the tooltip content is just in this table (excel spreadsheet also attached in comments for folks without CDC access, go to Tooltip sheet to see tooltips): [Tooltip table](https://cdc.sharepoint.com/:x:/r/teams/OPHDST-IRD-DIBBs/Shared%20Documents/CDC%20DIBBs%20Project/Streamline%20Team/1%20-%20Pilot%20eCR%20Viewer%202024/5%20-%20Research%20%26%20Design/Design/eCR%20Viewer%20Field%20Mapping.xlsx?d=w80a53c04215840b6a7762dfc8b7267c9&csf=1&web=1&e=X2ZJDc&nav=MTVfe0M2RUYzOTY4LURCMzMtNDIzNS05NUE0LUMyNzU1MDExQjE4Nn0)

[USWDS documentation for tooltips](https://designsystem.digital.gov/components/tooltip/)

Please use the "show on bottom" setting to start, and we'll see if that looks good in design review!

### PR DESCRIPTION
**Pull request by nickbristow**
# PULL REQUEST

## Summary
Integrates tool tips throughout the ECR viewer. See ticket for specific text

## Related Issue
Fixes #1478

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Comment by nickbristow** [comment]
@sarahtress 
image
image
image
image
image
image

**Comment by sarahtress** [comment]
@nickbristow looking good! a few things:

- the tooltips for Miscellaneous Notes and Jurisdiction Sent eCR to have bold text, can that be changed to match the rest?
- the tooltip for Jurisdiction Sent eCR is below the field name, can it be above like the rest?
- there should also be a tool tip on Reportable Condition and RCKMS Rule Summary in the Reportability Summary Table (these tooltips are slightly different from the ones in the eCR Summary section) and also a tooltip on the Sender Software field, can I see screenshots of those if they have been added? and if they weren't added can they be added?
- this one is more minor, but it looks like the tooltips are centered on the whole space that the Field Names take up, which looks odd when the name is short (see screenshot of Patient IDs below. is it possible to center the tooltips on the actual text of the Field Names?


Screenshot 2024-05-03 at 1 55 05 PM

**Comment by aviars** [comment]
Some copytext isn't a complete sentence. Otherwise rendering issue looks fixed.

**Comment by aviars** [comment]
@nickbristow  How about "A Patient ID is a unique identifier for a patient. This field normally contains a social security number or other medical record number."

**Comment by sarahtress** [comment]
> @nickbristow For PatientID, how about "A Patient ID is a unique identifier for a patient. This field normally contains a social security number or other medical record number."?

@aviars we already had our content designer review all of the tooltips. it was an intentional choice to keep them as incomplete sentences! please don't make any changes to the copy of the tooltips - thanks!

File: containers/ecr-viewer/src/app/view-data/components/UnavailableInfo.tsx [hunk 1]
```diff
@@ -41,8 +43,10 @@ const UnavailableInfo: React.FC<UnavailableInfoProps> = ({
   vitalUnavailableData,
   treatmentData,
   clinicalNotesData,
+  ecrMetadataUnavailableData,
 }) => {
-  const renderSection = (sectionTitle: string, data: DisplayData[]) => {
+  console.log("ecrMetadataUnavailableData", ecrMetadataUnavailableData);
```
Comment (BobanL) [inline containers/ecr-viewer/src/app/view-data/components/UnavailableInfo.tsx:48]: Can we remove this log?

File: containers/ecr-viewer/src/styles/custom-styles.scss [hunk 2]
```diff
@@ -288,3 +288,22 @@ ul.usa-sidenav > li.usa-sidenav__item > ul.usa-sidenav__sublist > li.usa-sidenav
...
+  white-space: normal !important;
+  text-align: left !important;
+  font-weight: normal !important;
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.short-tooltip + .usa-tooltip__body {
+  min-width: 350px !important;
+}
```
Comment (BobanL) [inline containers/ecr-viewer/src/styles/custom-styles.scss:309]: Do we still need !important here? I tested locally with the `!important` removed and these properties were still being used.

**Review by BobanL — COMMENTED** [review]
A couple of nits but looks great!

File: containers/ecr-viewer/src/app/view-data/components/UnavailableInfo.tsx (hunk 1 above)
Comment (nickbristow) [reply containers/ecr-viewer/src/app/view-data/components/UnavailableInfo.tsx:48]: ill miss it

**Review by BobanL — APPROVED** [review]
🌮

**Pull request merged**